.github/workflows/step_build.yml

```diff
@@ -0,0 +1,64 @@
+name: build
+run-name: Build test
+
+on:
+  workflow_call:
+
+concurrency:
+  group: ${{ github.ref }}
```

Comment:
What do you mean about `concurrency deadlock`? What issues have you encountered?

Reply:
I think I was using same group -> https://github.com/mahendrapaipuri/jupytext/actions/runs/6547300550